packages/matter.js/src/cluster/definitions/TemperatureMeasurementCluster.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*** THIS FILE IS GENERATED, DO NOT EDIT ***/
+
+import { Cluster, Attribute, OptionalAttribute } from "../../cluster/Cluster.js";
+import { MatterApplicationClusterSpecificationV1_1 } from "../../spec/Specifications.js";
+import { TlvInt16, TlvUInt16 } from "../../tlv/TlvNumber.js";
+import { TlvNullable } from "../../tlv/TlvNullable.js";
+
+/**
+ * Temperature Measurement
+ *
+ * This cluster provides an interface to temperature measurement functionality, including configuration and provision
+ * of notifications of temperature measurements.
+ *
+ * @see {@link MatterApplicationClusterSpecificationV1_1} § 2.3
+ */
+export const TemperatureMeasurementCluster = Cluster({
+    id: 0x402,
+    name: "TemperatureMeasurement",
+    revision: 4,
+
+    attributes: {
+        /**
+         * Represents the temperature in degrees Celsius as follows:
+         *
+         * MeasuredValue = 100 x temperature [°C]
+         *
+         * Where -273.15°C ≤ temperature ≤ 327.67°C, with a resolution of 0.01°C. The null value indicates that the
+         * temperature is unknown.
+         *
+         * @see {@link MatterApplicationClusterSpecificationV1_1} § 2.3.4.1
+         */
+        measuredValue: Attribute(0x0, TlvNullable(TlvInt16)),
```

Comment:
formally min: -27315 (indirectly in specs)

Reply:
Picked up by logic added earlier to transfer min/max bounds from fields referenced in the constraint definition